charts/backstage/templates/dynamic-plugins-configmap.yaml

```diff
@@ -33,4 +33,4 @@ type: Opaque
 stringData:
   .npmrc: |
     @redhat:registry=https://npm.registry.redhat.com
-{{- end }}
\ No newline at end of file
+{{- end }}
```

Comment:
You should be able to remove this Secret then, if the direct HTTP URL works..

Reply:
yes, you have to remove it I guess